.github/workflows/quality.yml

```
@@ -83,6 +90,12 @@
         with:
           fetch-depth: 0  # Shallow clones should be disabled for better relevancy
       
+      - name: Download coverage report
+        uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)